hclsyntax/parse_traversal_test.go

```diff
@@ -226,5 +279,26 @@ func TestParseTraversalAbs(t *testing.T) {
 				}
 			}
 		})
+
+		t.Run(fmt.Sprintf("partial_%s", test.src), func(t *testing.T) {
+			if test.src == "foo[*].bar" {
+				// Skip the test that's supposed to fail for splat syntax.
+				t.Skip("skipping test for unsupported splat syntax")
```

Comment:
Why don't we test for this to error?

Reply:
ah, this is a copy-paste error which makes the message wrong. This test case is expecting an error, which is not what we want here. So I skip this test case in the test branch that does support splats, otherwise the test will fail expecting an error when there isn't one.

Reply:
I'll fix this shortly!